contracts/interchain-token-service/InterchainTokenService.sol

```diff
@@ -678,11 +676,11 @@ contract InterchainTokenService is
         address tokenAddress = getStandardizedTokenAddress(tokenId);
         address tokenManagerAddress = getTokenManagerAddress(tokenId);
         address distributor = distributorBytes.length > 0 ? distributorBytes.toAddress() : tokenManagerAddress;
+        if (distributor == address(0)) revert ZeroAddress();
```

Comment:
user can pass 0 for the distributor on the origin if they don't want to give mint/burn permission to anyone other than token manager.

Also, `distributor` is being passed as `mintTo` below. StandardizedToken should check if `mintTo` is `0` and skip minting in that case (i think rn it'll fail).